pkg/cmd/infra/router/template.go

```diff
@@ -77,6 +78,7 @@ func (o *TemplateRouter) Bind(flag *pflag.FlagSet) {
 	flag.StringVar(&o.WorkingDir, "working-dir", "/var/lib/haproxy/router", "The working directory for the router plugin")
 	flag.StringVar(&o.DefaultCertificate, "default-certificate", util.Env("DEFAULT_CERTIFICATE", ""), "The contents of a default certificate to use for routes that don't expose a TLS server cert; in PEM format")
 	flag.StringVar(&o.DefaultCertificatePath, "default-certificate-path", util.Env("DEFAULT_CERTIFICATE_PATH", ""), "A path to default certificate to use for routes that don't expose a TLS server cert; in PEM format")
+	flag.StringVar(&o.DefaultCertificateDir, "default-certificate-dir", util.Env("DEFAULT_CERTIFICATE_DIR", ""), "A path to secret that contains the default certificate; may be in PEM format")
```

Comment:
How about: "A path to a directory that contains two files named tls.crt and tls.key, containing the certificate and key respectively.  These are combined at run time to make the PEM haproxy needs."

Although... is it better for us to just concatenate all of the files in that directory to make the PEM and ignore the specific names?  @DirectXMan12 what do you think?